pebblo/app/service/local_ui/loader_apps.py

```diff
@@ -379,11 +420,11 @@ def get_loader_app_details(self, db, app_name):
             logger.debug(message)
             return json.dumps(report_data, default=str, indent=4)
 
-    def _create_report_summary(self, raw_data, app_data):
+    @staticmethod
+    def _create_report_summary(raw_data, app_data):
```

Comment:
Lets add function param type and output type

Reply:
Added.